<ui version="4.0" >
 <class>uoReportPrevDlg</class>
 <widget class="QDialog" name="uoReportPrevDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <property name="horizontalSpacing" >
    <number>1</number>
   </property>
   <property name="verticalSpacing" >
    <number>1</number>
   </property>
   <item rowspan="4" row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>2</number>
     </property>
     <property name="verticalSpacing" >
      <number>2</number>
     </property>
     <item row="0" column="1" >
      <widget class="QPushButton" name="m_pbPrint" >
       <property name="text" >
        <string>Print...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="m_pbPage" >
       <property name="text" >
        <string>Page...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="7" >
      <widget class="QPushButton" name="m_pbZoomIn" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>ZoomIn</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="8" >
      <widget class="QPushButton" name="m_pbZoomOut" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>ZoomOut</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="9" >
      <widget class="QPushButton" name="m_pbClose" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="10" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6" >
      <widget class="QPushButton" name="m_pbNext" >
       <property name="text" >
        <string>Next</string>
       </property>
       <property name="icon" >
        <iconset resource="../uoReport.qrc" >:/image/go-next.PNG</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="m_pbPrevious" >
       <property name="text" >
        <string>Prev.</string>
       </property>
       <property name="icon" >
        <iconset resource="../uoReport.qrc" >:/image/go-previous.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="1" column="0" colspan="11" >
      <widget class="QFrame" name="m_frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QLineEdit" name="m_leCurentPage" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="inputMask" >
        <string>99999; </string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="maxLength" >
        <number>5</number>
       </property>
       <property name="cursorPosition" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5" >
      <widget class="QLabel" name="m_labelTotalPg" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>/ 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pbPrint</tabstop>
  <tabstop>m_pbPage</tabstop>
  <tabstop>m_pbNext</tabstop>
  <tabstop>m_pbPrevious</tabstop>
  <tabstop>m_pbZoomIn</tabstop>
  <tabstop>m_pbZoomOut</tabstop>
  <tabstop>m_pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="../uoReport.qrc" />
 </resources>
 <connections/>
</ui>
